drivers/gps/nrf9160_gps/nrf9160_gps.c

```diff
@@ -68,7 +68,7 @@ struct nrf9160_gps_config {
 	bool priority;
 };
 
-static int stop_gps(struct device *dev, bool is_timeout);
+static int stop_gps(const struct device *dev, bool is_timeout);
```

Comment:
Missing the const at the stop() function https://github.com/nrfconnect/sdk-nrf/blob/1f4578bf152f5d3c36442f6deeccb3e0a9fb83b8/drivers/gps/nrf9160_gps/nrf9160_gps.c#L724